config/test/test.config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     moduleNameMapper: {
         "^@khanacademy/perseus(.*)$":
             "<rootDir>/packages/perseus$1/src/index.js",
-        "^@khanacademy/kas$": "<rootDir>/packages/kas/kas.js",
+        "^@khanacademy/kas$": "<rootDir>/packages/kas/src/index.js",
```

Comment:
Same. Can we just say that all directories within `/packages` are actually packages?

Reply:
Similar concerns as with .flowconfig.